tests/test_properties.py

```diff
@@ -29,3 +30,23 @@ class B(enum.Enum):
 
     with pytest.raises(AttributeError, match=r"B.b has no attribute 'val'"):
         B.b.val
+
+
+def test_django_choices():
```

Comment:
Note that `TextChoices` can also be instantiated without explicit labels. This is valid:

```python
class D(TextChoices):
    a = "a"
```